src/ontology/uberon.Makefile

```diff
@@ -1101,11 +1097,24 @@ COLLECTED_vertebrate_SOURCES =       $(COLLECTED_tetrapod_SOURCES) \
 COLLECTED_metazoan_SOURCES =         $(COLLECTED_vertebrate_SOURCES) \
 				     $(COLLECTED_drosophila_SOURCES) \
 				     $(COLLECTED_worm_SOURCES) \
-				     $(IMPORTDIR)/local-ssso.owl \
+				     $(IMPORTDIR)/local-sslso.owl \
+				     $(BRIDGEDIR)/uberon-bridge-to-sslso.owl \
 				     $(IMPORTDIR)/local-ceph.owl \
 				     $(IMPORTDIR)/local-cteno.owl \
 				     $(IMPORTDIR)/local-poro.owl
 
+COLLECTED_lifestages_SOURCES =       $(SUBSETDIR)/life-stages-minimal.owl \
+				     $(IMPORTDIR)/local-fbdv.owl \
+				     $(IMPORTDIR)/local-wbls.owl \
+				     $(IMPORTDIR)/local-mmusdv.owl \
+				     $(IMPORTDIR)/local-hsapdv.owl \
+				     $(IMPORTDIR)/local-sslso.owl \
+				     $(BRIDGEDIR)/uberon-bridge-to-fbdv.owl \
+				     $(BRIDGEDIR)/uberon-bridge-to-wbls.owl \
+				     $(BRIDGEDIR)/uberon-bridge-to-mmusdv.owl \
+				     $(BRIDGEDIR)/uberon-bridge-to-hsapdv.owl \
+				     $(BRIDGEDIR)/uberon-bridge-to-sslso.owl
```

Comment:
Not actionable, just informative: I am noting that this seems to be missing a very important organism, Xenopus: https://www.ebi.ac.uk/ols4/ontologies/xao/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FXAO_1000000?lang=en

Reply:
Double-checking now, but it looks like XAO has indeed slipped through the cracks. It used to be provided by `ssso-merged`, so now we need to bring it ourselves like we do for, e.g., FBdv and WBls.

Reply:
XAO is a tricky case because, contrary to FBdv, WBls, and other similar species-specific ontologies, XAO contains both anatomical terms and life-stage terms, so we need to make a special case for this one (forcefully extracting a life-stage-only version of XAO).

Reply:
Done.